code/modules/uplink/uplink_items.dm

```diff
@@ -120,7 +120,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	name = "C-20r bundle"
 	desc = "Old faithful: The classic C-20r, bundled with two magazines, and a (surplus) suppressor at discount price."
 	item = /obj/item/storage/backpack/duffelbag/syndie/c20rbundle
-	cost = 14 // normally 16
+	cost = 13 // normally 16
```

Comment:
The C-20r is the bread and butter for nukeops. Why decrease the price when it's currently in a fairly good spot balance-wise?

Reply:
Fair fair